packages/pwa-kit-extension-sdk/src/shared/utils/resolver.ts

```diff
@@ -16,10 +16,10 @@ import {expand} from './index'
 import {BuildCandidatePathsOptions} from '../../types'
 
 // TODO: Should this be in a constants folder?
-const NODE_MODULES = 'node_modules'
-const OVERRIDES = 'overrides'
-const SRC = 'src'
-const APP = 'app'
+export const NODE_MODULES = 'node_modules'
+export const OVERRIDES = 'overrides'
+export const SRC = 'src'
+export const APP = 'app'
```

Comment:
If we have that constants file from my previous comment, we can use that here as well.